extensions/azurePublishNew/src/components/provisioningWizards/steps/ResourceConfigurationStep.tsx

```diff
@@ -0,0 +1,268 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React from 'react';
+import styled from '@emotion/styled';
+import { useRecoilState } from 'recoil';
+import formatMessage from 'format-message';
+import { FluentTheme, NeutralColors } from '@uifabric/fluent-theme';
+import {
+  ScrollablePane,
+  ScrollbarVisibility,
+  TooltipHost,
+  Icon,
+  Stack,
+  Text,
+  FontWeights,
+  FontSizes,
+  Label,
+  IStackTokens,
+  IStackItemStyles,
+  Link,
+} from 'office-ui-fabric-react';
+
+import {
+  userInfoState,
+  resourceGroupSelectionState,
+  subscriptionSelectionState,
+  tenantSelectionState,
+  deployLocationSelectionState,
+} from '../../../recoilModel/atoms/resourceConfigurationState';
+import { ResourceGroupPicker } from '../../resourceConfiguration/ResourceGroupPicker';
+import { SubscriptionPicker } from '../../resourceConfiguration/SubscriptionPicker';
+import { TenantPicker } from '../../resourceConfiguration/TenantPicker';
+import { DeployLocationPicker } from '../../resourceConfiguration/DeployLocationPicker';
+
+type Props = {
+  onResourceConfigurationChange: (isValidConfiguration: boolean) => void;
+};
+
+const ConfigureResourcesSectionName = styled(Text)`
+  font-size: ${FluentTheme.fonts.mediumPlus.fontSize};
+  font-weight: ${FontWeights.semibold};
+  margin-bottom: 4px;
+`;
+
+const ConfigureResourcesSectionDescription = styled(Text)`
+  font-size: ${FluentTheme.fonts.medium.fontSize};
```

Comment:
why is this medium font then line-height is fontsize 14?

Reply:
I double checked, it's not needed.